### PR TITLE
Wfnews 860

### PR DESCRIPTION
**Pull request by yzlucas**
also trigger searchTextUpdated after deleting keyword from location searchbox

### CODE REVIEW

**Review by dhlevi — APPROVED** [review]

File: client/wfnews-war/src/main/angular/src/app/components/wildfires-list/wildfires-list.component.ts [hunk 1]
```diff
@@ -271,4 +271,23 @@ export class WildFiresListComponent extends CollectionComponent implements OnCha
...
+  }
+
+  isSearchText() {
+    if (this.searchText && this.searchText !== ""){
+      return true
+    }
+  }
+
+  locationNameUpdated() {
+    if (this.locationName == '') {
```
Comment (dhlevi) [inline client/wfnews-war/src/main/angular/src/app/components/wildfires-list/wildfires-list.component.ts:287]: ===

File: client/wfnews-war/src/main/angular/src/app/components/wildfires-list/wildfires-list.component.ts [hunk 2]
```diff
@@ -271,4 +271,23 @@ export class WildFiresListComponent extends CollectionComponent implements OnCha
...
   }
 
+  isLocationName() {
+    if (this.locationName && this.locationName !== ""){
+      return true
+    }
+  }
+
+  isSearchText() {
+    if (this.searchText && this.searchText !== ""){
```
Comment (dhlevi) [inline client/wfnews-war/src/main/angular/src/app/components/wildfires-list/wildfires-list.component.ts:281]: consistent with the quotes. Use single or double, but try not to mix both

File: client/wfnews-war/src/main/angular/src/app/components/wildfires-list/wildfires-list.component.ts [hunk 3]
```diff
@@ -271,4 +271,23 @@ export class WildFiresListComponent extends CollectionComponent implements OnCha
     this.doSearch()
   }
 
+  isLocationName() {
+    if (this.locationName && this.locationName !== ""){
```
Comment (dhlevi) [inline client/wfnews-war/src/main/angular/src/app/components/wildfires-list/wildfires-list.component.ts:275]: just return the conditional:

`return this.locationName && this.locationName !== ''`

File: client/wfnews-war/src/main/angular/src/app/components/wildfires-list/wildfires-list.component.ts [hunk 4]
```diff
@@ -271,4 +271,23 @@ export class WildFiresListComponent extends CollectionComponent implements OnCha
...
+
+  isSearchText() {
+    if (this.searchText && this.searchText !== ""){
+      return true
+    }
+  }
+
+  locationNameUpdated() {
+    if (this.locationName == '') {
+      this.selectedLat=undefined;
```
Comment (dhlevi) [inline client/wfnews-war/src/main/angular/src/app/components/wildfires-list/wildfires-list.component.ts:288]: I don't mind if you don't use semi-colons (not required in typescript outside of specific cases) but don't mix them up or it starts to look weird. Also, it's nice to format things so there's spaces between symbols

`this.selectedLong = undefined` is more readable

File: client/wfnews-war/src/main/angular/src/app/components/wildfires-list/wildfires-list.component.ts (hunk 3 above)
Comment (yzlucas) [reply client/wfnews-war/src/main/angular/src/app/components/wildfires-list/wildfires-list.component.ts]: return this.locationName && this.locationName !== ''  doesn't really work here.
when this.locationName = '', return ( this.locationName && this.locationName !== '' ) will not return a boolean. 
It is actually return ( '' & false) , and the result is not false

Comment (dhlevi) [reply client/wfnews-war/src/main/angular/src/app/components/wildfires-list/wildfires-list.component.ts]: Right. You'd have to define the conditional to get rid of the behaviour when using `if (this.locationName)`

**Pull request merged**